src/main/java/nl/knaw/dans/bridge/plugin/lib/bagit/BagInfoCompleter.java

```diff
@@ -0,0 +1,43 @@
+package nl.knaw.dans.bridge.plugin.lib.bagit;
```

Comment:
This is never used, only potentially by clients of this library. Also it feels more like a 'util' because it's Bagit specific. Guess we are better of if you move it to the 'util' package.